Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    superlogger (1.0.1)
+    superlogger (1.0.2)
```

Comment:
Apparently building the gem also bumps this 🙈 Will take note of this for the next release.

Reply:
Suggestion: How about adding release instructions to the `README`? (Doesn't have to be part of this PR)

Reply:
The proper solution would honestly be a CD job because there are other aspects. I looked inside the gemfile and noticed that all of the temporary files from running the test cases were also bundled inside, which is why the gem increased in size from the previous version.

Reply:
I'll do that as a stopgap measure since it doesn't make too much sense to put too much effort here, thanks!

Reply:
Added in https://github.com/moexmen/superlogger/pull/26

Reply:
Thank you!

> I looked inside the gemfile and noticed that all of the temporary files from running the test cases were also bundled inside, which is why the gem increased in size from the previous version - a CD job would resolve this.

🙈